File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/Chart/AntennaGainItem.cs
namespace Kendo.Mvc.Examples.Models.Chart
{
    public class AntennaGainItem
    {
        public double Azimuth { get; set; }
        public double Gain { get; set; }

        public AntennaGainItem(double azimuth, double gain)
        {
            Azimuth = azimuth;
            Gain = gain;
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/AntennaGainItem.java
package com.kendoui.spring.models;

public class AntennaGainItem {
    private double azimuth;
    private double gain;
    
    public AntennaGainItem(double azimuth, double gain) {
        setAzimuth(azimuth);
        setGain(gain);
    }
    
    public double getAzimuth() {
        return azimuth;
    }

    public void setAzimuth(double azimuth) {
        this.azimuth = azimuth;
    }
    
    public double getGain() {
        return gain;
    }

    public void setGain(double gain) {
        this.gain = gain;
    }
}
